src/HotChocolate/AspNetCore/src/AspNetCore/Serialization/DefaultHttpRequestParser.cs

```diff
@@ -47,6 +51,88 @@ internal class DefaultHttpRequestParser : IHttpRequestParser
             CancellationToken cancellationToken) =>
             ReadAsync(stream, false, cancellationToken);
 
+        public async ValueTask<IReadOnlyList<GraphQLRequest>> ReadMultipartRequestAsync(
```

Comment:
We should replace httpRequest with form.

The middleware should be responsible for dealing with the request and the feature.